tasks/main.yml

```diff
@@ -14,8 +14,8 @@
 - name: Create a virtualenv with latest pip.
   pip: name=pip
        virtualenv={{ helga_home }}
+       virtualenv_command={{ virtualenv_bin }}
        extra_args='--upgrade'
-       virtualenv_python=python2
```

Comment:
Keep this line, so it knows which python interpreter to use. the virtualenv command is the one that is installed by default with python3.4, so its going to try to make a virtualenv with python3.4, but helga is python2 only